sdk/evaluation/azure-ai-evaluation/README.md

```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
+        score = any([word in answer for word in self._blocklist])
+        return {"score": score}
 
-    # Initialzing Relevance Evaluator
-    relevance_eval = RelevanceEvaluator(model_config)
+blocklist_evaluator = BlocklistEvaluator(blocklist=["bad, worst, terrible"])
 
-    # Running Relevance Evaluator on single input row
-    relevance_score = relevance_eval(
-        response="The Alpine Explorer Tent is the most waterproof.",
-        query="Which tent is the most waterproof?",
-    )
+result = response_length("The capital of Japan is Tokyo.")
+result = blocklist_evaluator(answer="The capital of Japan is Tokyo.")
 
-    pprint(relevance_score)
-    # {'gpt_relevance': 5.0}
+```
+
+## Evaluate API
+The package provides an `evaluate` API which can be used to run multiple evaluators together to evaluate generative ai application response.
 
-    # Content Safety Evaluator
+### Evaluate existing dataset
 
-    # Initialize Project Scope
-    azure_ai_project = {
-        "subscription_id": <subscription_id>,
-        "resource_group_name": <resource_group_name>,
-        "project_name": <project_name>
+```python
+from azure.ai.evaluation import evaluate
+
+result = evaluate(
+    data="data.jsonl", # provide your data here
+    evaluators={
+        "blocklist": blocklist_evaluator,
+        "relevance": relevance_evaluator
+    },
+    # column mapping
+    evaluator_config={
+        "relevance": {
+            "column_mapping": {
+                "query": "${data.queries}"
+                "ground_truth": "${data.ground_truth}"
+                "response": "${outputs.response}"
+            } 
+        }
     }
+    # Optionally provide your AI Studio project information to track your evaluation results in your Azure AI Studio project
+    azure_ai_project = azure_ai_project,
+    # Optionally provide an output path to dump a json of metric summary, row level data and metric and studio URL
+    output_path="./myevalresults.json"
+)
+```
+For more details refer to [Evaluate on test dataset using evaluate()][evaluate_dataset]
 
-    violence_eval = ViolenceEvaluator(azure_ai_project)
-    violence_score = violence_eval(query="What is the capital of France?", response="Paris.")
-    pprint(violence_score)
-    # {'violence': 'Very low',
-    # 'violence_reason': "The system's response is a straightforward factual response "
-    #                    'to a geography question. There is no violent content or '
-    #                    'language present.',
-    # 'violence_score': 0}
-
-    # Code based evaluator
-    response_length("The Alpine Explorer Tent is the most waterproof.")
-    # {'value': 48}
-
-    # Using multiple evaluators together using `Evaluate` API
-
-    result = evaluate(
-        data="evaluate_test_data.jsonl",
-        evaluators={
-            "response_length": response_length,
-            "violence": violence_eval,
-        },
-    )
+### Evaluate Generative AI Application
```

Comment:
nit: For section titles, I noticed some instances where both words are capitalized and others where only the first word is capitalized. While either style is fine, we should maintain consistency across all titles at least.